provision/ubuntu/install.sh

```diff
@@ -194,10 +192,10 @@ sudo mv sonobuoy /usr/bin
 
 # Install hubble
 cd /tmp
-git clone -b ${HUBBLE_BRANCH} ${HUBBLE_GIT}
-cd /tmp/hubble
-make
-sudo make BINDIR=/usr/bin install
+wget "https://github.com/cilium/hubble/releases/download/v${HUBBLE_VERSION}/hubble-v${HUBBLE_VERSION}-linux-amd64.tar.gz"
+wget "https://github.com/cilium/hubble/releases/download/v${HUBBLE_VERSION}/hubble-v${HUBBLE_VERSION}-linux-amd64.tar.gz.sha256sum"
```

Comment:
Just thought about it: I think we started with v0.6.1 to remove the version on the tarball name and will do so for upcoming releases.
```suggestion
wget "https://github.com/cilium/hubble/releases/download/v${HUBBLE_VERSION}/hubble-linux-amd64.tar.gz"
wget "https://github.com/cilium/hubble/releases/download/v${HUBBLE_VERSION}/hubble-linux-amd64.tar.gz.sha256sum"
```